### PR TITLE
Fix flowcells not getting correct status `ondisk` after retrieval from PDC

### PR DESCRIPTION
**Pull request by barrystokman**
This PR adds/fixes that after being retrieved from PDC, flowcells don't get their status updated from 'processing'
 to 'ondisk'. Flowcells were stuck on `processing`, blocking the automation from retrieving the next requested flowcell(s).

### How to prepare for test
- [x] ssh to hasta (depending on type of change)
- [x] install on stage: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh flowcell-ondisk-fix`
- [x] install on stage: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-servers-stage.sh backup_config`
- [x] activate stage `us`
****
#### TC1: dry run
- [x] run `cg backup fetch-flowcell -f H2YC5CCXX --dry-run`
#### TC2: real run
- [x] run `cg backup fetch-flowcell -f H2YC5CCXX`
****
#### TC1: dry run
- [x] check that correct retrieval command is printed to terminal: `ssh nas-9.scilifelab.se bash SCRIPTS/retrieve_run_nas.bash H2YC5CCXX hasta /home/proj/stage/flowcells/hiseqx/`
- [x] check that the retrieval time is printed to terminal, as indication that the code finished running succesfully: `Retrieval time: n.nnnh` where n.nnn can be any number.
#### TC2: real run
- [x] log in as `hiseq.clinical`
- [x] preconditions: before running the command, verify that that flowcell has the correct status `removed`: `cg get flowcell H2YC5CCXX --no-samples`
- [x] now run the command
- [x] verify that that flowcell has the correct status `processing`: `cg get flowcell H2YC5CCXX --no-samples`
- [x] check that correct retrieval command is preinted to terminal: `ssh nas-9.scilifelab.se bash SCRIPTS/retrieve_run_nas.bash H2YC5CCXX hasta /home/proj/stage/flowcells/hiseqx/`
- [x] check that the retrieval process has actually started
- [x] check that the retrieval time is printed to terminal, as indication that the code finished running succesfully: `Retrieval time: n.nnnh` where n.nnn can be any number.
- [x] check that the flowcell is now on hasta in the directory `home/proj/stage/flowcells`
- [x]  verify that that flowcell has the correct status `ondisk`: `cg get flowcell H2YC5CCXX --no-samples`
****
## Review
- [ ] code approved by
- [x] tests executed by @barrystokman 
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!
****
### This [version](https://semver.org/) is a
- [x] **MINOR** - when you add functionality in a backwards compatible manner

To be deployed together with https://github.com/Clinical-Genomics/servers/pull/424

### CODE REVIEW

**Comment by barrystokman** [comment]
TC1:
image
:heavy_check_mark: 

TC2:
preconditions:
image
:heavy_check_mark:

**Comment by barrystokman** [comment]
image
Verify status is set to `preocessing` during retrieval :heavy_check_mark:

File: cg/meta/backup.py [hunk 1]
```diff
@@ -55,24 +56,36 @@ def fetch_flowcell(self, flowcell_obj: models.Flowcell = None, dry_run: bool = F
             LOG.info("maximum flowcells ondisk reached")
             return None
 
+        if flowcell_obj:
+            flowcell_obj.status = "processing"
+            if not dry_run:
+                self.status.commit()
+
         if not flowcell_obj:
```
Comment (moonso) [inline cg/meta/backup.py:64]:
```suggestion
        else:
```

File: cg/meta/backup.py [hunk 2]
```diff
@@ -55,24 +56,36 @@ def fetch_flowcell(self, flowcell_obj: models.Flowcell = None, dry_run: bool = F
...
             if flowcell_obj is None:
                 LOG.info("no flowcells requested")
                 return None
 
         if not dry_run:
-            LOG.info(f"{flowcell_obj.name}: retrieving from PDC")
+            LOG.info("%s: retrieving from PDC", flowcell_obj.name)
 
-        tic = time.time()
+        start_time = time.time()
```
Comment (moonso) [inline cg/meta/backup.py:73]: 👍 thank you for that

**Review by moonso — APPROVED** [review]
Nice job @barrystokman !

**Comment by barrystokman** [comment]
Bumped:
image
Deployed:

**Pull request merged**